vmdb/spec/lib/miq_automation_engine/miq_ae_state_machine_new_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+require 'timecop'
+
+describe "MiqAeStateMachine" do
+  before do
+    DummyWorkspace = Class.new do
+      def initialize(options = {})
```

Comment:
@mkanoor  With the change @kbrock is suggesting the `after` block to remove the class constants is no longer needed.